### PR TITLE
TaskTest: implement FloatingTaskTest

### PR DESCRIPTION
**Pull request by nightism**

### CODE REVIEW

**Comment by nightism** [comment]
Just realize that FloatingTaskTest was not implemented.

File: src/test/java/seedu/address/model/task/FloatingTaskTest.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import org.junit.Before;
+import org.junit.Test;
+
+public class FloatingTaskTest {
+    private FloatingTask floatingTask;
+
+    @Before
+    public void setupFloatingTask() throws Exception {
+        final Name name = new Name("Floating Task Name");
+        Priority priority = new Priority("3");
```
Comment (pyokagan) [inline src/test/java/seedu/address/model/task/FloatingTaskTest.java:16]: Why is this not final?

**Review by pyokagan — APPROVED** [review]
Thanks. I'll make the single tweak and merge.

**Comment by pyokagan** [comment]
Merged as  ecd657a3dab96915737fb20495925e6f0a9cde83

**Pull request closed without merging**